spec/support/api/schemas/case_cycle.json

```diff
@@ -0,0 +1,33 @@
+{
+  "type": "object",
+  "required": [
+    "created_at",
+    "id",
+    "slug",
+    "status",
+    "submitted_on",
+    "updated_at",
+    "user_id"
+  ],
+  "properties": {
+    "case_number": { "type":  "string"},
+    "copay_cents": { "type": "integer", "minimum": 0 },
+    "copay_currency": { "$ref": "currency_or_null.json" },
+    "copay_frequency": {
+      "anyOf": [
+        { "$ref":"copay_frequencies.json" },
+        { "type": "null" }
+      ]
+    },
+    "created_at": { "type": "string", "format": "date-time" },
+    "effective_on": { "$ref": "currency_or_null.json" },
```

Comment:
Should this be currency?  Looks like it should be a date.

Reply:
yup - definitely a _date_